apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -132,45 +137,73 @@ const styles = {
   description: {
     paddingRight: 10,
     paddingBottom: 10,
-    fontSize: 16,
-    fontFamily: 'Gotham 3r',
-    zIndex: 2,
+    fontSize: 14,
+    fontFamily: '"Gotham 5r"',
     color: color.charcoal,
     width: 940
   }
 };
 
-const NewProjectButtons = React.createClass({
-  propTypes: {
+const TILES_PER_ROW = 4;
+
+class NewProjectButtons extends React.Component {
+  static propTypes = {
     projectTypes: PropTypes.arrayOf(PropTypes.string),
-    isRtl: PropTypes.bool
-  },
+    isRtl: PropTypes.bool,
+    description: PropTypes.string,
+  };
 
   render() {
-    const { isRtl } = this.props;
-    const thumbnailStyle = isRtl ? styles.thumbnailRtl : styles.thumbnail;
+    const { isRtl, description } = this.props;
     // Using absolute URLs to get this working in storybook.
     const projectTypes = this.props.projectTypes || DEFAULT_PROJECT_TYPES;
     return (
       <div style={styles.fullsize}>
-        <div style={styles.description}>{i18n.projectStartNew()}</div>
-        <div>
-          {
-            projectTypes.slice(0,4).map((projectType, index) => (
-              <a key={index} href={"/projects/" + projectType + "/new"}>
-                <div style={[styles.tile, index < 3 && styles.tilePadding]}>
-                  <img style={thumbnailStyle} src={PROJECT_INFO[projectType].thumbnail} />
-                  <div style={styles.label}>
-                    {PROJECT_INFO[projectType].label}
-                  </div>
-                </div>
-              </a>
-            ))
-          }
-        </div>
+        {description && <div style={styles.description}>{description}</div>}
+        {
+          _.chunk(projectTypes, TILES_PER_ROW).map((projectTypesRow, rowIndex) => (
+              <NewProjectButtonRow
+                key={rowIndex}
+                isRtl={isRtl}
+                projectTypes={projectTypesRow}
+              />
+            )
+          )
+        }
       </div>
     );
   }
-});
+}
 
 export default Radium(NewProjectButtons);
+
+class NewProjectButtonRowUnwrapped extends React.Component {
```

Comment:
Why do we need to have a new row component? Can't they just wrap with CSS?

Reply:
Originally I had extracted this component because I thought I needed to write code to instantiate it multiple times, but now that I'm using `_.chunk` there is only one invocation so I agree it's better to keep it inline.